setup.py

```diff
@@ -45,6 +45,7 @@
         'semantic-version',
         'jsonschema>=3.2.0',
         'google-cloud-storage',
+        'python-ipmi',
```

Comment:
remember to send an email to OCS-QE regarding the update to the venv before once we merge the PR

Reply:
Sure will do